fastlane/lib/fastlane/actions/appaloosa.rb

```diff
@@ -104,6 +104,7 @@ def self.upload_on_appaloosa(api_key, store_id, binary_path, screenshots, group_
         req.body = { store_id: store_id,
                      api_key: api_key,
                      mobile_application_update: {
+                       description: description,
```

Comment:
hey @ohwutup thank you for your time.
i read the current api v2 https://appaloosa-store.com/api and the description field is optional.